functions/_pure_prompt_npmrc.fish

```diff
@@ -0,0 +1,12 @@
+function _pure_prompt_npmrc --description "Display npmrc profile"
+    if set --query pure_enable_npmrc; and test "$pure_enable_npmrc" = true
+        set --local npmrc ''
+        set --local npmrc_color (_pure_set_color $pure_color_npmrc)
+        if test -e "$HOME/.npmrc"
+            set npmrc (readlink $HOME/.npmrc | cut -d '/' -f 5)
```

Comment:
`readlink` has no output when `$HOME/.npmrc` is a regular file and not a symlink. You don't need to resolve the link just to show the basename.

```suggestion
            set npmrc (basename $HOME/.npmrc)
```